lib/identity_cache/query_api.rb

```diff
@@ -100,6 +100,10 @@ def expire_primary_key_cache_index(id)
         IdentityCache.cache.delete(rails_cache_key(id))
       end
 
+      def cached_association(name) # :nodoc:
```

Comment:
We can also add a comment like `# @api private` on top just to make it clear.

Reply:
Yeah, I always found `:nodoc:` to be kind of implicit about its meaning

Reply:
TIL 👍